configuration/micro_kernel_trait.rst

```diff
@@ -245,6 +274,46 @@ because the configuration started to get bigger:
             ;
         };
 
+As well as the ``config/web_profiler.yaml`` file:
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # config/web_profiler.yaml
+        when@dev:
+            web_profiler:
+                toolbar: true
+                intercept_redirects: false
+
+    .. code-block:: xml
+
+        <!-- config/web_profiler.xml -->
+        <?xml version="1.0" encoding="UTF-8" ?>
+        <container xmlns="http://symfony.com/schema/dic/services"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xmlns:framework="http://symfony.com/schema/dic/symfony"
+            xsi:schemaLocation="http://symfony.com/schema/dic/services https://symfony.com/schema/dic/services/services-1.0.xsd
+                http://symfony.com/schema/dic/symfony https://symfony.com/schema/dic/symfony/symfony-1.0.xsd">
+
+            <!-- WebProfilerBundle is enabled only in the "dev" environment -->
+            <when env="dev">
+                <web-profiler:config toolbar="true" intercept-redirects="false"/>
+            </when>
+        </container>
+
+    .. code-block:: php
+
+        // config/web_profiler.php
+        use Symfony\Config\WebProfilerConfig;
+
+        return static function (WebProfilerConfig $webProfiler) {
+            $webProfiler
```

Comment:
Missing `env` check

Reply:
https://github.com/symfony/symfony-docs/issues/15223#issuecomment-1126023101 😞

Reply:
Reverted according to the previous comment.